src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/wrapped/AbstractWrappedRequestManager.java

```diff
@@ -122,7 +123,7 @@ public IToken reassignRequest(@NotNull final IToken token, @NotNull final Collec
      */
     @NotNull
     @Override
-    public <T extends IRequestable> IRequest<T> getRequestForToken(@NotNull final IToken token) throws IllegalArgumentException
+    public IRequest<?> getRequestForToken(@NotNull final IToken<?> token) throws IllegalArgumentException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)